int32 id
int32 startNode
int32 destNode
float32 weight
